src/widget/wfindonmenu.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <QAction>
+#include <QMenu>
+#include <QUrlQuery>
+
+class Track;
+
+class WFindOnMenu : public QMenu {
+    Q_OBJECT
+  public:
+    explicit WFindOnMenu(
+            QWidget* parent = nullptr);
+    ~WFindOnMenu() override = default;
+
+    enum class Services {
+        SoundCloud,
+        LastFm,
+        Discogs
+    };
+
+    void openTheBrowser(const QString& serviceUrl,
+            const QString& query);
+
+    void createAllServices(const Track& track);
```

Comment:
"create all services"? I don't understand what this method is supposed to do.

Reply:
I agree, without a comment, it looks like this function created the whole service itself. 
How about createSubmenusForAllWebLookups() or such.  


By the way, in header we use /// comments that are picked up by doxygen and asigned to the following function which is able to automatically create an API decryption along with a class hierarchy.